pages/common/xargs.md

```diff
@@ -9,7 +9,7 @@
 
 - Delete all files with a `.backup` extension:
```

Comment:
What difference is the `--delimiter` option making here ? The output from `find` is already delimited by newlines.

Reply:
It's changing the delimeter on `xargs` not `find`. By default `xargs` uses any whitespace as its delimiter.